### ISSUE TITLE
PHP 8

### ISSUE THREAD

**Issue by 021fneves** [HansSchouten/PHPageBuilder#183]
Does PHPageBuilder work with PHP 8?

**Comment by bipalam** [comment]
I don't think so. I am on 8+ and I get this error when I try to create a new page - 
`Deprecated: Creation of dynamic property PHPageBuilder\PageTranslation::$id is deprecated in /var/www/vhosts/xxxxx.com/httpdocs/phpagebuilder/src/Repositories/BaseRepository.php on line 236`

**Comment by webffo** [comment]
The page builder is working, that's just a deprecation warning. You can hide it by either
`error_reporting(E_ALL & ~E_DEPRECATED);`

or by 
`ini_set('display_errors', 0);`
